tests/unit/test_slot_enumerations.cairo

```diff
@@ -172,22 +172,14 @@ func test_token_index_reverts_if_index_invalid{
     return ();
 }
 
-@view
-func test_token_index_zero_nonexistent{
```

Comment:
This should now test that SafeMath does its job when using the index `Uint256(2**128-1, 2**128-1)` 
(-1 as Uint256)